packaging/pre_build_script.sh

```diff
@@ -17,4 +17,4 @@ else
 fi
 pip install $PYTORCH_DEP
 
-pip install setuptools wheel twine auditwheel
+pip install setuptools wheel twine auditwheel numpy
```

Comment:
`numpy` was added to DTensor here: https://github.com/pytorch/pytorch/pull/165451, so now we need to add it to the environment for successful wheel builds